src/cdk-experimental/popover-edit/table-directives.ts

```diff
@@ -374,11 +374,11 @@ export class CdkRowHoverContent implements AfterViewInit, OnDestroy {
                 // Not doing any positioning in CDK version. Material version
                 // will absolutely position on right edge of cell.
                 this.viewRef = this.viewContainerRef.createEmbeddedView(this.templateRef, {});
-                this.initElement(this.elementRef.nativeElement!.nextSibling as HTMLElement);
+                this.initElement(this.viewRef.rootNodes[0] as HTMLElement);
               } else {
                 this.viewContainerRef.insert(this.viewRef);
               }
-              this.prepareElement(this.elementRef.nativeElement!.nextSibling as HTMLElement);
+              this.prepareElement(this.viewRef.rootNodes[0] as HTMLElement);
```

Comment:
To be clear: The logic right now only supports the first element in the embedded view. This change just makes it work with Ivy.. there is no change in behavior, but we should revisit handling all root nodes in the popover template.

cc. @kseamon

Reply:
Looks reasonable.

Out of curiosity, what would be the circumstance that there'd be more than one node? Maybe if the directive was attached to ng-container?

Reply:
Developers could do something like below. It's probably an edge-case but might be still good to handle as it can cause confusing behavior and this is valid unwrapping of the micro syntax.

```html
<ng-template matRowHoverContent>
  <span>Text 1</span>
  <span>Text 2</span>
</ng-template>
```